.github/workflows/sdk-issue.yaml

```diff
@@ -0,0 +1,12 @@
+name: Create Jira Task
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  call-create-jira-task:
+    uses: maxio-com/ab-java-sdk/.github/workflows/sdk-issue-template.yaml@main
+    with:
+      sdk-name: "Golang SDK"
```

Comment:
You can add: `AB Golang SDK` to be consistent with other repositories.

Reply:
other dont have ab prefix